src/components/JsonSchemaValidator/dbtUnified.js

```diff
@@ -0,0 +1,409 @@
+    snowplow__enable_mobile_context: {
+      type: 'boolean',
+      title: 'Enable Mobile Context',
+    },
+    snowplow__enable_geolocation_context: {
+      type: 'boolean',
+      title: 'Enable Geolocation Context',
+    },
+    snowplow__enable_application_context: {
+      type: 'boolean',
+      title: 'Enable App Context',
+    },
+    snowplow__enable_screen_context: {
+      type: 'boolean',
+      title: 'Enable Screen Context',
+    },
+    snowplow__enable_app_errors: {
+      type: 'boolean',
+      title: 'Enable App Error Context',
+    },
+    snowplow__enable_deep_link_context: {
+      type: 'boolean',
+      title: 'Enable Deep Link Context',
+    },
+    snowplow__enable_browser_context: {
+      type: 'boolean',
+      title: 'Enable Browser Context',
+    },
+    snowplow__has_log_enabled: {
+      type: 'boolean',
+      title: 'Enable Run Logs',
+    },
+    snowplow__ua_bot_filter: {
+      type: 'boolean',
+      title: 'Filter Bots',
+    },
+    snowplow__databricks_catalog: {
+      type: 'string',
+      title: '(Databricks) Catalog',
+      description: 'The catalogue your atomic events table is in',
+    },
+    snowplow__page_view_context: {
+      type: 'string',
+      title: '(Redshift) Page View Context Table',
+    },
+    snowplow__iab_context: {
+      type: 'string',
+      title: '(Redshift) iab Context Table',
+    },
+    snowplow__ua_parser_context: {
+      type: 'string',
+      title: '(Redshift) UA Context Table',
+    },
+    snowplow__yauaa_context: {
+      type: 'string',
+      title: '(Redshift) YAUAA Context Table',
+    },
+    snowplow__cmp_visible_events: {
+      type: 'string',
+      title: '(Redshift) CMP Visible Context Table',
+    },
+    snowplow__consent_preferences_events: {
+      type: 'string',
+      title: '(Redshift) Consent Preferences Context Table',
+    },
+    snowplow__enable_load_tstamp: {
+      type: 'boolean',
+      title: '(Redshift) Enable load_tstamp',
+    },
+    snowplow__derived_tstamp_partitioned: {
+      type: 'boolean',
+      title: '(Bigquery) Dervied Timestamp Partition',
+    },
+
+    snowplow__ga4_categories_seed: {
+      type: 'string',
+      title: 'Seed reference for GA4 Categories',
+    },
+    snowplow__geo_mapping_seed: {
+      type: 'string',
+      title: 'Seed reference for geo mapping',
+    },
+    snowplow__rfc_5646_seed: {
+      type: 'string',
+      title: 'Seed reference for rfc 5646 (language mapping)',
+    },
+
+
+
+    snowplow__session_identifiers: {
+      type: 'string',
+      title: 'Session Identifiers',
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 0,
+      items: {
+          type: 'object',
+          title: "Identifier",
+          properties: {
+            schema: { type: 'string' }, // TODO: add regex here to make valid context/unstruct or atomic?
+            field: { type: 'string' } // TODO: add regex here to make valid SQL name?
+          },
```

Comment:
we should add some descriptions to these

Reply:
<img width="713" alt="image" src="https://github.com/snowplow/documentation/assets/95634439/f43548ae-a94f-444f-aa01-603c97589d9f">

is this still valid?

Reply:
for custom user identifiers

Reply:
Hm, @emielver do you remember if the identifiers can have a prefix argument? I assume for rs/pg?

Reply:
Yes looks like prefix is a valid field for both session AND user identifiers https://github.com/snowplow/dbt-snowplow-utils/blob/2eaea3ceed7f8636a0250088be9051e0273d44df/macros/base/base_create_snowplow_sessions_lifecycle_manifest.sql#L155

Reply:
added this too